### ISSUE TITLE
Desiger/dashboard internationalization

### ISSUE THREAD

**Issue by tomy2105** [elsa-workflows/elsa-core#1098]
First of all thanks for the good work, keep it up :)

Would be nice if there was a way to translate designer/dashboard into some other language easily using custom string definitions where all strings presented to user would be enumerated.

**Comment by sfmskywalker** [comment]
Yes, we need this.

**Comment by sfmskywalker** [comment]
Basic internationalization support has been implemented in the designer. But we also need it for the activity descriptors (C#). But I think we should do this as a separate issue. The Core project is getting too big for my taste, so I would like to clean that up before adding more to it. Probably as part of the 3.0 milestone where certain aspects will be reconsidered anyway.

**Comment by sfmskywalker** [comment]
I just realized that the current implementation doesn't easily scale. Right now, it depends on PRs coming in to update the translations for each component. Over time, this will increase the bundle size. Better would be to have separate translation bundles that one can optionally install.

**Comment by tomy2105** [comment]
Yes, good idea.... to have separate translation bundles.
In addition, probably can be done already but just mentioning just in case and searching for documnetation how to do it....
No has to be able to write his own translation bundle and just somehow register it with the "core UI"....

**Comment by sfmskywalker** [comment]
Moving this forward with #1359
